modules/coursier/shared/src/main/scala/coursier/parse/ReconciliationParser.scala

```diff
@@ -0,0 +1,34 @@
+package coursier.parse
+
+import coursier.core.{Module, ModuleName, Organization, Reconciliation}
+import coursier.util.{ModuleMatcher, ModuleMatchers, ValidationNel}
+import coursier.util.Traverse._
+
+object ReconciliationParser {
+  def reconciliation(input: Seq[String], scalaVersionOrDefault: String): Either[List[String], Seq[(ModuleMatchers, Reconciliation)]] = {
+    DependencyParser.moduleVersions(
+      input, scalaVersionOrDefault
+    ).either match {
+      case Left(e) => Left(e)
+      case Right(elems) =>
+        (elems.validationNelTraverse { case (m: Module, v: String) =>
+          val e = reconciliation(m, v)
+          ValidationNel.fromEither(e)
+        }).either
+    }
+  }
+
+  def reconciliation(module: Module, v: String): Either[String, (ModuleMatchers, Reconciliation)] = {
```

Comment:
Make that method `private`?

Reply:
Addressed in https://github.com/coursier/coursier/pull/1293/commits/7d8966f262200bff55a5bef70b19af11af0f7931